migrations/versions/0299_org_types_table.py

```diff
@@ -0,0 +1,45 @@
+"""
+
+Revision ID: 0299_org_types_table
+Revises: 0298_add_mou_signed_receipt
+Create Date: 2019-07-10 16:07:22.019759
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = '0299_org_types_table'
+down_revision = '0298_add_mou_signed_receipt'
+
+
+def upgrade():
+    organisation_types_table = op.create_table(
+        'organisation_types',
+        sa.Column('name', sa.String(), nullable=False),
+        sa.PrimaryKeyConstraint('name'),
+        sa.Column('is_crown', sa.Boolean, nullable=True),
+        sa.Column('annual_free_sms_fragment_limit', sa.BigInteger, nullable=False)
+
+    )
+
+    op.bulk_insert(
+        organisation_types_table,
+        [
+            {'name': x, 'is_crown': y, 'annual_free_sms_fragment_limit': z} for x, y, z in [
+                ["central", None, 250000],
```

Comment:
do we need to add nhs?

Reply:
I don't think we do, since we don't have a constraint on the table at this point, and in next PR I will migrate all 'nhs' organisation_type organisations to be either 'nhs_local' or 'nhs_central' before introducing the constraint.

Reply:
Actually you are right that this is cleaner code, I added it.